source/Graphite.System/EventlogListener.cs

```diff
@@ -119,8 +125,19 @@ private void OnEntryWritten(object sender, EntryWrittenEventArgs e)
             if (!string.IsNullOrEmpty(this.category) && !this.category.Equals(e.Entry.Category, StringComparison.OrdinalIgnoreCase))
                 return;
 
+            if (this.eventid.HasValue && !this.eventid.Equals(e.Entry.EventID))
```

Comment:
`EventID` seems to be obsolete (see https://msdn.microsoft.com/en-us/library/system.diagnostics.eventlogentry.eventid.aspx). Could you change it to `InstanceId`? I think it should have the same meaning.